src/webui/www/public/login.html

```diff
@@ -6,60 +6,23 @@
     <title>qBittorrent QBT_TR(Web UI)QBT_TR[CONTEXT=OptionsDialog]</title>
     <link rel="icon" type="image/png" href="images/skin/qbittorrent32.png" />
     <link rel="stylesheet" type="text/css" href="css/login.css?v=${VERSION}" />
-    <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
-    <script>
-        'use strict';
-
-        window.onload = function() {
-            $('username').focus();
-            $('username').select();
-        };
-
-        window.addEvent('domready', function() {
-            $('loginform').addEvent('submit', function(e) {
-                new Event(e).stop();
-                submitLoginForm();
-            });
-        });
-
-        function submitLoginForm() {
-            new Request({
-                url: 'api/v2/auth/login',
-                method: 'post',
-                data: $('loginform').toQueryString(),
-                onComplete: function() {
-                    $('password').set('value', '');
-                },
-                onFailure: function(xhr) {
-                    if (xhr.responseText !== "") {
-                        $('error_msg').set('html', xhr.responseText);
-                    }
-                    else {
-                        $('error_msg').set('html', 'QBT_TR(Unable to log in, qBittorrent is probably unreachable.)QBT_TR[CONTEXT=HttpServer]');
-                    }
-                },
-                onSuccess: function(text) {
-                    if (text == "Ok.") {
-                        // Session started. Simply reload page.
-                        window.location.reload();
-                    }
-                    else {
-                        $('error_msg').set('html', 'QBT_TR(Invalid Username or Password.)QBT_TR[CONTEXT=HttpServer]');
-                    }
-                }
-            }).send();
-        }
-    </script>
+    <noscript>
+        <link rel="stylesheet" type="text/css" href="css/noscript.css?v=${VERSION}" />
+    </noscript>
+    <script src="scripts/login.js?locale=${LANG}&v=${VERSION}"></script>
```

Comment:
@Chocobo1:  Can this line be written like this? 

```
<script src="scripts/login.js?locale=${LANG}&v=${VERSION}" />

Reply:
I think better not:
https://stackoverflow.com/a/8853550